tools/authoring-decisions.adoc

```diff
@@ -307,9 +311,18 @@ The **Decision Service** node needs to be selected in order to see its context m
 It is not enough to hover over the **Decision Service** node, as it is for other DMN node types.
 
 
-.Unkown node
+===== Unknown node
+
+Is a node with an invalid DMN element reference(`dmnElementRef`).
+We can not render such node properly as we miss details about the node.
+For such nodes editor users will see a placeholder node.
 
-. TODO
+image::tools/authoringDecisions/new-editor-unknown-node.png[]
+
+**(1) Reference** value is the `dmnElementRef` value that was not possible to resolve to a node.
+The editor does not provide a feature to change this value.
+To change this value open and edit the file manually outside the editor.
+Other possible fix is to include a model that contains a node with `id` equal to given `dmnElementRef` value or to define such a node in the current model.
```

Comment:
**(1) Reference** is the `dmnElementRef` value that could not resolve to a node.
The editor does not provide a feature to change this value.
To change this value, open and edit the file manually outside the editor.
Other possible fixes are including a model that contains a node with `id` equal to the given `dmnElementRef` value or defining such a node in the current model.